ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -84,6 +84,7 @@ Initialize Global Variables
     Set Global Variable   ${RHODS_VERSION}
     Set Prometheus Variables
     Set Global Variable    ${DASHBOARD_APP_NAME}    ${PRODUCT.lower()}-dashboard
+    Set Global Variable    ${RHODS_PACKAGE_NAME}    rhods-operator
```

Comment:
are you working on passing this variable from CI/CD job? otherwise ODH uninstall won't work

Reply:
as far as I can see we have the same package name also for ODH:

```
➜  oc get sub -A                                                                                                             
NAMESPACE               NAME                  PACKAGE               SOURCE                CHANNEL
konveyor-forklift       forklift-operator     forklift-operator     community-operators   stable-v2.6
opendatahub-operators   rhoai-operator-dev    rhods-operator        rhoai-catalog-dev     odh-nightlies
...
```